src/Core_Screencap/Core.ScreenshotManager.cs

```diff
@@ -194,6 +203,21 @@ private void InitializeSettings()
                 "UI Screenshots", "Screenshot resolution multiplier",
                 1,
                 new ConfigDescription("Multiplies the UI screenshot resolution from the current game resolution by this amount.\nWarning: Some elements will still be rendered at the original resolution (most notably the interface).", new AcceptableValueRange<int>(1, 8), "Advanced"));
+
+            // config presets
+            presetCount = Config.Bind(
+                "Presets", "Preset Count",
+                1,
+                new ConfigDescription("Amount of additional screenshot presets. Requires Restart to apply.", new AcceptableValueRange<int>(0, 10), "Advanced"));
+
+
+            for(int i = 0; i < presetCount.Value; i++)
+            {
+                presetKeys.Add(Config.Bind("Presets", $"~{i + 1} - Shortcut", KeyboardShortcut.Empty, new ConfigDescription($"Keyboard Shortcut for Preset #{i + 1}.", null, "Advanced")));
+                presetResX.Add(Config.Bind("Presets", $"~{i + 1} - Res Horizontal", Screen.width, new ConfigDescription($"Horizontal size (width) of rendered screenshots in pixels of Preset #{i + 1}.", new AcceptableValueRange<int>(ScreenshotSizeMin, ScreenshotSizeMax), "Advanced")));
+                presetResY.Add(Config.Bind("Presets", $"~{i + 1} - Res Vertical", Screen.height, new ConfigDescription($"Vertical size (height) of rendered screenshots in pixels of Preset #{i + 1}.", new AcceptableValueRange<int>(ScreenshotSizeMin, ScreenshotSizeMax), "Advanced")));
+                presetDownscaling.Add(Config.Bind("Presets", $"~{i + 1} - Upsampling Ratio", 2, new ConfigDescription($"Capture screenshots in a higher resolution and then downscale them to desired size for Preset #{i + 1}. Prevents aliasing, perserves small details and gives a smoother result, but takes longer to create.", new AcceptableValueRange<int>(1, 4), "Advanced")));
```

Comment:
Wouldn't it make more sense to include number of the preset in the category name? e.g. `[Custom Hotkey 1]`

[Here's](https://github.com/BepInEx/BepInEx.Utility.IL2CPP/blob/main/Patch_ByteFiddlerIL2CPP_Net6/ByteFiddlerPatcher.cs#L22-L45) how I implemented something similar, a bit less user friendly though.